services/venice-router/src/main/java/com/linkedin/venice/router/VeniceRouterConfig.java

```diff
@@ -277,7 +277,7 @@ private void checkProperties(VeniceProperties props) {
     perNodeClientAllocationEnabled = props.getBoolean(ROUTER_PER_NODE_CLIENT_ENABLED, false);
     perNodeClientThreadCount = props.getInt(ROUTER_PER_NODE_CLIENT_THREAD_COUNT, 2);
 
-    keyValueProfilingEnabled = props.getBoolean(KEY_VALUE_PROFILING_ENABLED, false);
+    keyValueProfilingEnabled = props.getBoolean(KEY_VALUE_PROFILING_ENABLED, true);
```

Comment:
I don't mind changing the default here... with the optimizations you did, it should outweigh the cost of this metric. But please mention it in your commit message.

Reply:
Oh no, this is for testing purpose. I don't think we should enable it by default. May be I can create a task for rolling out k/v profiling in prod slowly or as needed basis.